Networking/Networking/Model/ShippingLabel/Packages/Custom package/WooShippingCustomPackage.swift

```diff
@@ -31,10 +31,10 @@ public struct WooShippingCustomPackage: Equatable, GeneratedFakeable {
         PackageType(rawValue: rawType) ?? .box
     }
 
-    public init(id: String, name: String, type: String, dimensions: String, boxWeight: Double) {
+    public init(id: String, name: String, rawType: String, dimensions: String, boxWeight: Double) {
```

Comment:
Should we maybe have all dimensions as separate properties?
Since the API has them like that https://github.com/woocommerce/woocommerce-shipping/blob/trunk/docs/api/packages.md

That is why I had it like that for the `WooShippingPackageDataRepresentable` "dummy" data phase.